<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QDialog" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>293</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Qwit</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../qwit.qrc" >
    <normaloff>:/images/qwit.png</normaloff>:/images/qwit.png</iconset>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <layout class="QVBoxLayout" >
     <item>
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QLabel" name="greetingMessageLabel" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font" >
          <font>
           <family>Arial</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text" >
          <string>What are you doing?</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="charsLeftLabel" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="palette" >
          <palette>
           <active>
            <colorrole role="WindowText" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>0</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Light" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>0</red>
               <green>170</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Dark" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>170</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Text" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>0</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="ButtonText" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>0</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="WindowText" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>0</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Light" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>0</red>
               <green>170</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Dark" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>170</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Text" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>0</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="ButtonText" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>0</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled>
            <colorrole role="WindowText" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>170</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Light" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>0</red>
               <green>170</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Dark" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>170</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Text" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>170</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="ButtonText" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>170</red>
               <green>0</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
           </disabled>
          </palette>
         </property>
         <property name="font" >
          <font>
           <family>Arial</family>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="toolTip" >
          <string>Left characters count - please, do not exceed this number</string>
         </property>
         <property name="frameShape" >
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow" >
          <enum>QFrame::Plain</enum>
         </property>
         <property name="text" >
          <string>140</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget" >
       <property name="toolTip" >
        <string/>
       </property>
       <property name="tabPosition" >
        <enum>QTabWidget::South</enum>
       </property>
       <property name="currentIndex" >
        <number>0</number>
       </property>
       <widget class="QWidget" name="homeTab" >
        <attribute name="title" >
         <string>Home</string>
        </attribute>
        <attribute name="toolTip" >
         <string>Your and your friends' timeline</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="publicTab" >
        <attribute name="title" >
         <string>Public</string>
        </attribute>
        <attribute name="toolTip" >
         <string>Public timeline</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="repliesTab" >
        <attribute name="title" >
         <string>Replies</string>
        </attribute>
        <attribute name="toolTip" >
         <string>Replies to you</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="customTab" >
        <attribute name="title" >
         <string>Custom</string>
        </attribute>
        <attribute name="toolTip" >
         <string>Custom user timeline</string>
        </attribute>
        <layout class="QGridLayout" >
         <item row="0" column="0" >
          <layout class="QVBoxLayout" >
           <item>
            <layout class="QHBoxLayout" >
             <item>
              <widget class="QLabel" name="label_2" >
               <property name="text" >
                <string>Username:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="customUsernameLineEdit" />
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="inboxTab" >
        <attribute name="title" >
         <string>Inbox</string>
        </attribute>
        <attribute name="toolTip" >
         <string>Direct messages inbox</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="outboxTab" >
        <attribute name="title" >
         <string>Outbox</string>
        </attribute>
        <attribute name="toolTip" >
         <string>Direct messages outbox</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="searchTab" >
        <attribute name="title" >
         <string>Search</string>
        </attribute>
        <attribute name="toolTip" >
         <string>Search for messages</string>
        </attribute>
        <layout class="QGridLayout" >
         <item row="0" column="0" >
          <layout class="QVBoxLayout" >
           <item>
            <layout class="QHBoxLayout" >
             <item>
              <widget class="QLabel" name="label_3" >
               <property name="text" >
                <string>Search:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="searchLineEdit" />
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="spacing" >
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="stateLabel" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Ignored" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape" >
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow" >
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="text" >
          <string/>
         </property>
         <property name="indent" >
          <number>-1</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="refreshPushButton" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip" >
          <string>Refresh</string>
         </property>
         <property name="text" >
          <string/>
         </property>
         <property name="icon" >
          <iconset resource="../qwit.qrc" >
           <normaloff>:/images/reload.png</normaloff>:/images/reload.png</iconset>
         </property>
         <property name="shortcut" >
          <string>Ctrl+R</string>
         </property>
         <property name="autoDefault" >
          <bool>false</bool>
         </property>
         <property name="flat" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="optionsPushButton" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip" >
          <string>Options</string>
         </property>
         <property name="text" >
          <string/>
         </property>
         <property name="icon" >
          <iconset resource="../qwit.qrc" >
           <normaloff>:/images/options.png</normaloff>:/images/options.png</iconset>
         </property>
         <property name="autoDefault" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="logsPushButton" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string/>
         </property>
         <property name="icon" >
          <iconset resource="../qwit.qrc" >
           <normaloff>:/images/logs.png</normaloff>:/images/logs.png</iconset>
         </property>
         <property name="autoDefault" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="exitPushButton" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip" >
          <string>Quit</string>
         </property>
         <property name="text" >
          <string/>
         </property>
         <property name="icon" >
          <iconset resource="../qwit.qrc" >
           <normaloff>:/images/exit.png</normaloff>:/images/exit.png</iconset>
         </property>
         <property name="autoDefault" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../qwit.qrc" />
  <include location="../qwit.qrc" />
 </resources>
 <connections/>
</ui>
